web/src/components/core/IssuesAlert.tsx

```diff
@@ -35,7 +37,14 @@ export default function IssuesAlert({ issues }) {
     >
       <List>
         {issues.map((i: Issue, idx: number) => (
-          <ListItem key={idx}>{i.description}</ListItem>
+          <ListItem key={idx}>
+            {i.description}{" "}
+            {i.kind === "solver" && (
+              <Link to={PATHS.conflicts} variant="link" isInline>
+                {_("Review and fix")}
```

Comment:
Please add some context and a hint for the translators:

```suggestion
                // TRANSLATORS: Clickable link to show and resolve package dependency conflicts
                {_("Review and fix")}
```